src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/SID.cs

```diff
@@ -423,13 +417,11 @@ private void CreateFromBinaryForm(byte[] binaryForm, int offset)
 
             if (binaryForm.Length - offset < SecurityIdentifier.MinBinaryLength)
             {
-                throw new ArgumentOutOfRangeException(
-nameof(binaryForm),
-                    SR.ArgumentOutOfRange_ArrayTooSmall);
+                throw new ArgumentOutOfRangeException(nameof(binaryForm), SR.ArgumentOutOfRange_ArrayTooSmall);
             }
 
-            IdentifierAuthority Authority;
-            int[] SubAuthorities;
+            IdentifierAuthority authority;
```

Comment:
Does this need to be defined up here, or can it instead be moved down to where it's assigned about 30 lines later?

Reply:
Moved.